tests/Unit/Utilities/Test_Formaline.cpp

```diff
@@ -10,6 +10,7 @@
 #include "Utilities/Formaline.hpp"
 
 SPECTRE_TEST_CASE("Unit.Utilities.Formaline", "[Unit][Utilities]") {
```

Comment:
Add a `CHECK(true)` or similar to this test to make newer Catch happy on OSX.

Reply:
yes, that is one of the test failures...
No assertions in test case 'Unit.Utilities.Formaline'

Reply:
The other is in Unit.IO.H5.File
```

############ ERROR ############
Node: 0 Proc: 0
Line: 473 of /Users/kidder/Codes/SpECTRE/nils/spectre/src/IO/H5/Helpers.cpp
Function: T h5::read_data(const hid_t, const std::string &) [Rank = 1, T = std::__1::vector<char, std::__1::allocator<char> >]
At least one element in 'size' is 0. Expected data along 1 dimensions. size = (0)
############ ERROR ############


------------- Processor 0 Exiting: Called CmiAbort ------------
Reason: 
[0] Stack Traceback:
  [0:0] 0   RunTests                            0x0000000111d2f82b CmiAbortHelper + 187
  [0:1] 1   RunTests                            0x0000000111d2cc6b CmiAbort + 43
  [0:2] 2   RunTests                            0x000000010006fe5a _ZN8Parallel5abortERKNSt3__112basic_stringIcNS0_11char_traitsIcEENS0_9allocatorIcEEEE + 218
  [0:3] 3   RunTests                            0x0000000111bf724a _Z24abort_with_error_messagePKciS0_RKNSt3__112basic_stringIcNS1_11char_traitsIcEENS1_9allocatorIcEEEE + 1674
  [0:4] 4   RunTests                            0x000000010e447eae _ZN2h59read_dataILm1ENSt3__16vectorIcNS1_9allocatorIcEEEEEET0_xRKNS1_12basic_stringIcNS1_11char_traitsIcEES4_EE + 2686
  [0:5] 5   RunTests                            0x000000010e47a275 _ZN2h513SourceArchiveC2EbONS_6detail9OpenGroupExRKNSt3__112basic_stringIcNS4_11char_traitsIcEENS4_9allocatorIcEEEE + 1365
  [0:6] 6   RunTests                            0x000000010e47aa0d _ZN2h513SourceArchiveC1EbONS_6detail9OpenGroupExRKNSt3__112basic_stringIcNS4_11char_traitsIcEENS4_9allocatorIcEEEE + 61
  [0:7] 7   RunTests                            0x0000000108c2a01d _ZNK2h56H5FileILNS_10AccessTypeE0EE3getINS_13SourceArchiveEJEEERKT_RKNSt3__112basic_stringIcNS8_11char_traitsIcEENS8_9allocatorIcEEEEDpOT0_ + 4365
  [0:8] 8   RunTests                            0x0000000108c03ca9 _ZZL30____C_A_T_C_H____T_E_S_T____17vENK3$_1clIN2h56H5FileILNS1_10AccessTypeE0EEEEEDaRKT_ + 201
  [0:9] 9   RunTests                            0x0000000108be9b9e _ZL30____C_A_T_C_H____T_E_S_T____17v + 734

```